responses/configuration/set_test.go

```diff
@@ -11,11 +11,9 @@ func TestGetValidation(t *testing.T) {
 		up          SetConfigResponse
 		expectError bool
 	}{
-		{"valid   - all parameters (Code, Description,ExpectedValues) proper", SetConfigResponse{Code: "Logging.EnableRemote", Description: "true", ExpectedValues: []string{"abc", "def"}}, false},
-		{"invalid - no parameter (Code, Description,ExpectedValues) proper", SetConfigResponse{Code: "", Description: "", ExpectedValues: nil}, true},
-		{"invalid - some parameters (Code, Description) proper, others not proper", SetConfigResponse{Code: "Logging.EnableRemote", Description: "false", ExpectedValues: nil}, true},
-		{"invalid - some parameters (Code, ExpectedValues) proper, others not proper", SetConfigResponse{Code: "Logging.EnableRemote", Description: "", ExpectedValues: []string{"abc", "def"}}, true},
-		{"invalid - some parameters (Description,ExpectedValues) proper, others not proper", SetConfigResponse{Code: "", Description: "true", ExpectedValues: []string{"abc", "def"}}, true},
+		{"valid   - all parameters (Success, Description) proper", SetConfigResponse{Success: true, Description: "Success"}, false},
+		{"valid   - all parameters (Success, Description) proper", SetConfigResponse{Success: false, Description: "Success"}, false},
+		{"invalid - one parameter (Success) proper, other not proper", SetConfigResponse{Success: true, Description: ""}, true},
```

Comment:
For completeness' sake, you might add an additional case where description is "" and success == false.